<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoinCourseForm</class>
 <widget class="QWidget" name="JoinCourseForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>JoinCourseFrom</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(200, 200, 200);
background-color: rgb(10, 10, 10);R</string>
  </property>
  <widget class="QLineEdit" name="course_id_input">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>270</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 2px solid rgb(45, 45, 45);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30, 30, 30);	
	color: rgb(100, 100, 100);
}
QLineEdit:hover {
	border: 2px solid rgb(55, 55, 55);
}
QLineEdit:focus {
	border: 2px solid rgb(255, 207, 0);	
	color: rgb(200, 200, 200);
}</string>
   </property>
   <property name="placeholderText">
    <string>请输入课堂号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>160</y>
     <width>220</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {	
	background-color: rgb(50, 50, 50);
	border: 2px solid rgb(60, 60, 60);
	border-radius: 5px;
}
QPushButton:hover {	
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(70, 70, 70);
}
QPushButton:pressed {	
	background-color: rgb(255, 144, 80);
	border: 2px solid rgb(255, 165, 24);	
	color: rgb(35, 35, 0);
}</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
